lib/Sema/PCMacro.cpp

```diff
@@ -282,6 +284,18 @@ class Instrumenter : InstrumenterBase {
     }
     return DCS;
   }
+  
+  DeferStmt *transformDeferStmt(DeferStmt *DS) {
+    if (auto *FD = DS->getTempDecl()) {
+      auto Implicit = FD->isImplicit();
+      FD->setImplicit(false);
```

Comment:
I don't love this hack, but I guess it's okay. Mind commenting *why* we have to do it, though?